pages/linux/lsfd.md

```diff
@@ -0,0 +1,20 @@
+# lsfd
+
+> Lists open files and the corresponding processes in Linux.
+> More information: <https://manned.org/lsfd>.
+
+- List all open file descriptors:
+
+`lsfd`
+
+- List all files kept open by a specific program:
+
+`lsfd -Q 'PID == {{process_ID}}'`
+
+- Check what program has a specific file open:
+
+`lsfd -Q "NAME == '/path/to/file'"`
```

Comment:
```suggestion
`lsfd -Q "NAME == '{{path/to/file}}'"`
```

Reply:
Absolute path is a must

Reply:
```suggestion
`lsfd -Q "NAME == '{{/path/to/file}}'"`
```

Okay, I still think that we should add a placeholder here.